src/fixit/config.py

```diff
@@ -226,20 +226,38 @@ def collect_rules(
             stack.enter_context(append_sys_path(path))
 
         for qualified_rule in config.enable:
-            rules = set(find_rules(qualified_rule))
-            if qualified_rule.name:
-                named_enables |= rules
-            all_rules |= rules
+            try:
+                rules = set(find_rules(qualified_rule))
+                if qualified_rule.name:
+                    named_enables |= rules
+                all_rules |= rules
+            except CollectionError as e:
+                log.warning(
+                    f"Collection Error: Failed to collect enabled rules for {config.enable}: {e}"
```

Comment:
Log messages should only print the specific `qualified_rule` that is actively trying to be found/imported, otherwise it may be difficult to track down the specific `enable` item that triggered the warning. Same for all the warnings below.

I think a simpler message like `Failed to load rules "fixit.rules": <exception>` would be better and easier to understand. We could probably combine the two except clauses and just use the latter form for everything.